.ci/jenkins/Jenkinsfile.buildchain

```diff
@@ -0,0 +1,111 @@
+@Library('jenkins-pipeline-shared-libraries')_
```

Comment:
btw, shouldn't that file moved to `kogito-pipelines` ???

Reply:
done by:

- https://github.com/kiegroup/optaplanner/pull/1440/commits/2d8bd042e1a8ac7236b4602c71d52f04d7cf12e8
- https://github.com/kiegroup/kogito-pipelines/pull/215/commits/04b49107d935fd2db7b1a00f6c2c3d120f616f0f